__tests__/__packages__/cli-test-utils/src/environment/doc/response/ITestEnvironment.ts

```diff
@@ -40,10 +29,10 @@ export interface ITestEnvironment<TestPropertiesSchema> {
      * in the case that no system test properties were configured or could be loaded.
      *
      * Not present if skipProperties is specified on ISetupEnvironmentParms
-     * @type {TestPropertiesSchema}
+     * @type {ITestPropertiesSchema}
      * @memberof ITestEnvironment
      */
-    systemTestProperties: TestPropertiesSchema;
```

Comment:
When using generics, it's advisable to keep the name short, and ideally something different to an existing interface.

you could get the same benefits by doing `<T extends ITestPropertiesSchema>` and use the value of `T` as the desired type for this properti (e.g. `systemTestProperties: T`)

Reply:
@awharn

Reply:
Since the value before was TestPropertiesSchema, do we need to make that change? I am also a bit confused with the difference between generics and interfaces in this particular context.

Reply:
Again, these are Typescript generics which do not have any impact on users.

Having the `ITestPropertiesSchema` will cause confusion as we might expect the interface to be taking effect when it won't.

The reason why `TestPropertiesSchema` never caused problems was because it was just the name that we gave it (which is equivalent to using a single letter, like `T`)

Hence the suggestion of `T extends ITestPropertiesSchema` to get the benefits of the interface 😋 

Feel free to ignore this comment if you feel I'm completely off base here (because, honestly, I might be) 😅

Reply:
ok i just reverted changes to the file... is that alright? commit: 2d1c9be60 
@zFernand0 @t1m0thyj